modules/javafx.base/src/test/java/test/JUnit5Test.java

```diff
@@ -25,12 +25,15 @@
 
 package test;
 
+import static org.junit.Assert.assertNotNull;
```

Comment:
For JUnit 5, shouldn't this be `org.junit.jupiter.api.Assertions.assertNotNull`?

Reply:
Sorry, I totally missed that.  Although you can mix these without ill effect, we should move towards the `jupiter` annotations if we ever want to get to the point where JUnit 4 can be phased out.